docs/usage/DirectMigrationExampleAPI.md

```diff
@@ -0,0 +1,200 @@
+#Direct Migration using API
+
+Please setup, deploy and configure the Migration Operator as well as the Migration Controller
+appropriately before proceeding further.
+
+We will be performing a migration of a Django Application consisting of persistent volumes and image streams, this
+application is already present on the source cluster used in the following migration example.
+
+The steps to perform a Direct Migration using Migration Toolkit for Containers (MTC) are as follows:
+
+**1. Add Migration clusters**<br>
+This step involves addition and configuration of the source as well as the destination clusters 
+involved in the migration.
+
+- At first, let's create a MigCluster instance in order to add the destination cluster. 
+In our case this cluster is also the host cluster where Migration controller resides and is
+already configured, you do not need to add this cluster. 
+
+Store the destination MigCluster yaml manifest in `destination-migcluster.yaml`:
+```
+apiVersion: migration.openshift.io/v1alpha1
+kind: MigCluster
+metadata:
+  name: host
+  namespace: openshift-migration
+spec:
+  isHostCluster: true
+```
+
+Command to create destination MigCluster instance:
+``` 
+oc create -f destination-migcluster.yaml
+```
+
+- Now let's add the second cluster, that is the source cluster.
+
+Store the source MigCluster yaml manifest in `source-migcluster.yaml`:
+``` 
+apiVersion: migration.openshift.io/v1alpha1
+kind: MigCluster
+metadata:
+  name: src-ocp-3-cluster
+  namespace: openshift-migration
+spec:
+  exposedRegistryPath: docker-registry-default.apps.shubham-ocp3.mg.dog8code.com
+  insecure: true
+  isHostCluster: false
+  serviceAccountSecretRef:
+    name: src-ocp-3-cluster-k2vnk
+    namespace: openshift-config
+  url: 'https://master.shubham-ocp3.mg.dog8code.com/'
```

Comment:
We should scrub this URL.

Reply:
Suggest:
```
  url: 'https://master.ocp3.mycluster.com/'
```

Reply:
Thanks for the suggestion @djwhatle , updated the PR.